### PR TITLE
Note detection WWWWWWWWWWWWWWW

### PR DESCRIPTION
**Pull request by JacksonElia**

### CODE REVIEW

**Comment by Ishan1522** [comment]
note detection w

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -295,9 +295,9 @@ public static final class FieldConstants {
   
   public static final class IntakeConstants {
     public static final int LEFT_INTAKE_MOTOR_ID = 18;
-    public static final int RIGHT_INTAKE_MOTOR_ID = 19;
+    public static final int RIGHT_INTAKE_MOTOR_ID = 2;
     public static final int FLAPPER_MOTOR_ID = 1;
-    public static final int NOTE_SENSOR_ID = 1; // TODO
+    public static final int NOTE_SENSOR_ID = 6; // TODO
```
Comment (JacksonElia) [inline src/main/java/frc/robot/Constants.java:300]: can we remove the todo?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -65,6 +70,7 @@ public RobotContainer() {
     intakeSubsystem = new IntakeSubsystem();
     pivotSubsystem = new PivotSubsystem();
     ledSubsystem = new LEDSubsystem();
+    noteDetector = new NoteDetector();
```
[the remark this answers is not in the document]
Comment (Ishan1522) [reply src/main/java/frc/robot/RobotContainer.java]: im pretty sure i did this

**Comment by Ishan1522** [comment]
should i merge dis @JacksonElia ?

**Pull request merged**